### PR TITLE
Let docs/api.md point to api/README.md

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by Hexcles** [comment]
Where's `docs/api.md` linked from? Or do you think this is a more natural place people might look for docs?

**Comment by foolip** [comment]
Yes, I just went looking for the API docs and couldn't find them :)

File: docs/api.md [hunk 1]
```diff
@@ -0,0 +1 @@
+See [api/README.md](api/README.md).
```
Comment (Hexcles) [inline docs/api.md]: The hyperlink doesn't work. Perhaps try `/api/README.md`?

Comment (foolip) [reply docs/api.md]: Ha, I had the slash, but didn't like how it looked, and removed both :)

**Pull request merged**